core-foundation-sys/src/base.rs

```diff
@@ -80,6 +80,23 @@ pub struct CFAllocatorContext {
     pub preferredSize: CFAllocatorPreferredSizeCallBack
 }
 
+/// Trait for all types which are Core Foundation reference types.
```

Comment:
I think this commit should be reordered to come before 3688803.

Reply:
Hmm. With "this commit" I assume you mean 0fc6b62f266203f3466e79d5c31a0b7cb91980f8 ? If so, it does come before 3688803